common/views/components/BasePage/EventPage.js

```diff
@@ -16,7 +16,7 @@ import {UiImage} from '../Images/Images';
 import type {UiEvent} from '../../../model/events';
 import {spacing, font} from '../../../utils/classnames';
 import camelize from '../../../utils/camelize';
-import {formatAndDedupeOnDate, formatAndDedupeOnTime, joinDateStrings, formatDayDate, isDatePast, formatTime} from '../../../utils/format-date';
+import {formatAndDedupeOnDate, formatAndDedupeOnTime, joinDateStrings, formatDayDate, isDatePast, isTimePast, formatTime} from '../../../utils/format-date';
```

Comment:
Could we drop these onto separate lines?
Do we want to think of having a maxlength lint rule?

Reply:
Looks like Prettier enforces newlines for multiple named imports. Possibly look to integrate it with eslint as well?